lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:dynflow_steps) do
+      if Sequel::Model.db.database_type == :mysql
+        set_column_type :data, :mediumtext
+      end
+    end
+  end
+
+  down do
+    if Sequel::Model.db.database_type == :mysql
+      set_column_type :data, :text
```

Comment:
Wouldn't this need to be wrapped in an `alter_table` block or given a table symbol directly?